### PR TITLE
fix(yarn-dev): Packages to devDependencies for yarn

### PR DESCRIPTION
**Pull request by rishabh3112**
Resolves: #31

### CODE REVIEW

File: generator.js [hunk 1]
```diff
@@ -189,6 +189,7 @@ module.exports = class WebpackGenerator extends Generator {
 
 	installPlugins() {
 		this.runInstall(getPackageManager(), this.dependencies, {
+			"dev": true,
```
Comment (sendilkumarn) [inline generator.js]: I don't think it is correct way to do this. Can we check the package manager?

Comment (rishabh3112) [reply generator.js]: Working on it @sendilkumarn.

File: generator.js [hunk 2]
```diff
@@ -188,9 +188,16 @@ module.exports = class WebpackGenerator extends Generator {
 	}
 
 	installPlugins() {
-		this.runInstall(getPackageManager(), this.dependencies, {
-			"save-dev": true
-		});
+		const pkgManager = getPackageManager();
```
[the remark this answers is not in the document]
Comment (rishabh3112) [reply generator.js]: Have you tested it on `npm` too?

[the remark this answers is not in the document]
Comment (sendilkumarn) [reply generator.js]: Instead of passing `yarn` and `npm` below use the package manager

**Comment by sendilkumarn** [comment]
Thanks @rishabh3112 again 👍

**Comment by rishabh3112** [comment]
@sendilkumarn  No need to thanks, I am just following my passion 😎.

**Pull request merged**